Assets/Scripts/Editor/PointCloud/Trees/PointImportJobs.cs

```diff
@@ -69,37 +72,45 @@ private struct ColorAccess
                         byte b = 0;
                         if (Color != null)
                         {
-                            var ptr = (byte*) Color + index * Stride;
+                            var ptr = (byte*)Color + index * Stride;
                             r = ptr[0];
                             g = ptr[1];
                             b = ptr[2];
-                            color = (uint) ((b << 16) | (g << 8) | r);
+                            color = (uint)((b << 16) | (g << 8) | r);
+                        }
+                        else if (ColorF != null)
+                        {
+                            var ptr = (byte*)ColorF + index * Stride;
```

Comment:
If field type is float then why this ptr is read as 3 bytes?

Reply:
As far as I know, I've only seen data in which 32bit is forcibly packed with 8bit rgb value.